layouts/shortcodes/blocks/feature.html

```diff
@@ -0,0 +1,10 @@
+{{ $icon := .Get "icon" | default "fa-lightbulb" }}
+{{ $url_text := .Get "url_text" }}
+<div class="col-lg-4 mb-5 mb-lg-0 text-center ">
+  <div class="mb-4 h1">
+    <i class="{{ if not (or (hasPrefix $icon "fas ") (hasPrefix $icon "fab ")) }}fas {{ end }}{{ $icon }}"></i>
+  </div>
+  <h4 class="h3">{{ .Get "title" | markdownify }}</h4>
+  <div class="mb-0">{{ .Inner }}</div>
```

Comment:
This is a `<p>` tag rather than a `<div>` in upstream. Overriding this file fixes it in the local project. Will upstream the change later.